src/reducers/reducer.js

```diff
@@ -1,5 +1,5 @@
-export default (type, initialData) => (state = {
-  loading: true,
+export default (type, initialData, initialLoading = true) => (state = {
```

Comment:
why is this needed?

Reply:
without this the table loading spinner is displaying before the user made the first query.

Reply:
maybe it should make an initial query by default so the page shows something?